langlib/langlib-test/src/test/resources/test-src/xmllib_test.bal

```diff
@@ -217,3 +217,52 @@ function testGet() returns [xml|error, xml|error, xml|error, xml|error, xml|erro
 
     return [e1, e2, c1, item, item2];
 }
+
+function testChildren() returns [[int, string, string, string, string], [int, string, string, string]] {
```

Comment:
Shall we move these test cases into Ballerina, as we are targetting to move away from testing in java classes.
You can use string comparisons and also comparisons with xml literals.

Reply:
+1, fixed.